pkg/kapp/cmd/appchange/time_flags.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package appchange
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+type TimeFlags struct {
+	Before     string
+	After      string
+	BeforeTime time.Time
+	AfterTime  time.Time
+
+	Duration string
+}
+
+func (t *TimeFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&t.Before, "before", "", "", "List app-changes happened before given time stamp (format: before=<timestamp>)")
```

Comment:
Can we also the timestamp format which is expected?

Reply:
Done